model_analyzer/monitor/gpu_free_memory.py

```diff
@@ -23,17 +23,43 @@
 # OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
```

Comment:
These are records, why did you move them from `records/`

Reply:
They are records, but they are associated with the monitor. I'll move them back I think that makes sense.